code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -530,6 +530,8 @@ datum
 				if (!iscarbon(M) && !ismobcritter(M))
 					return
 				if ((method == INGEST || (method == TOUCH && prob(25))) && (isdead(M) || istype(get_area(M),/area/afterlife/bar)))
+					if(volume <= 1)
+						return
```

Comment:
```suggestion
				if (volume < 1)
					return
				if ((method == INGEST || (method == TOUCH && prob(25))) && (isdead(M) || istype(get_area(M),/area/afterlife/bar)))
```
Changed the order, and also note the `<` instead of `<=` (as otherwise you would need more than 1u rather than it being the minimum).